src/FakeChatAppFunction.js

```diff
@@ -0,0 +1,52 @@
+import { useEffect, useState } from "react"
+
+const lessons = [
+  {
+    id: 1,
+    name: 'ReactJs la gi? Tai sao nen hoc ReactJS'
+  },
+  {
+    id: 2,
+    name: 'SPA, MPA la gi?'
+  },
+  {
+    id: 3,
+    name: 'Arrow Function'
+  },
+
+]
+
+const FakeChatAppFunction = () => {
+  const [lessonId, setLessonId] = useState(1)
+  
+
+  useEffect(() => {
+    const handleCmt = ({detail}) => {
+      console.log(detail)
+    }
+  
+  
+    window.addEventListener(`lession-${lessonId}`, handleCmt)
+
+    return ( () => {
+      window.removeEventListener(`lession-${lessonId}`, handleCmt)
+    })
+  })
+
+  
+
+  return (
+    <div>
```

Comment:
Please add className to `div` that we can find it easy.
if we only want to use `div` to warp the code => Please use `<></>`